packages/ra-ui-materialui/src/detail/CreateView.tsx

```diff
@@ -41,8 +40,15 @@ export const CreateView = (props: CreateViewProps) => {
     );
 };
 
-export type CreateViewProps = CreateProps;
-
+export interface CreateViewProps {
```

Comment:
I think it should extend the base div props (e.g. `style`), otherwise valid HTML attributes, that are passed down to the root components, con't be accepted. This also allows you to get rid of `className`. Same for the other 2.